.github/workflows/gh-ci.yaml

```
@@ -34,9 +34,14 @@ jobs:
           include:
             - name: macOS_monterey_py311
               os: macOS-12
-              python-version: "3.11"
+              python-version: "3.12"
               full-deps: true
               codecov: true
+            - name: macOS_M1
```

```suggestion
            - name: macOS_14_arm64_py312
```
Just adding a bit more detail for ease of recognition